doc/related_projects.rst

```diff
@@ -219,4 +245,39 @@ Domain specific packages
 Snippets and tidbits
 ---------------------
 
-The `wiki <https://github.com/scikit-learn/scikit-learn/wiki/Third-party-projects-and-code-snippets>`_ has more!
+**Gists**
```

Comment:
Yes, I was skeptic about including this here too. Should we then rename the original wiki to just "Code Snippets" ?